src/snowflake/snowpark/_internal/analyzer/analyzer.py

```diff
@@ -161,23 +161,29 @@ def analyze(
 
         if isinstance(expr, Like):
             return like_expression(
-                self.analyze(expr.expr, parse_local_name), self.analyze(expr.pattern)
+                self.analyze(expr.expr, parse_local_name),
+                self.analyze(expr.pattern, parse_local_name),
```

Comment:
This (and other changes in this PR) is already in the main branch. I'm wondering how it's shown as a change in this PR.